src/components/Dashboard/TodoList.vue

```diff
@@ -53,18 +53,16 @@
                 </v-btn>
               </h3>
 
-              <!-- BCOMP AR special case -->
-              <div v-if="businessId && isBComp && item.enabled && isTypeAnnualReport(item) && isStatusNew(item)"
-                class="list-item__subtitle pt-4"
-              >
+              <!-- Annual Report verify checbox -->
+              <div v-if="isShowAnnualReportCheckbox(item)" class="list-item__subtitle pt-4">
                 <p>Verify your Office Address and Current Directors before filing your Annual Report.</p>
                 <v-checkbox
                   id="enable-checkbox"
                   class="todo-list-checkbox"
                   label="All information about the Office Addresses and Current Directors is correct."
                   :disabled="!item.enabled || isCoaPending"
                   v-model="enableCheckbox[index]"
-                  @click.native.stop
+                  @click.stop
```

Comment:
This change worked, but I tried the same change in other places and, while the UI seemed to work fine, a number of unit tests broke, so I left those alone.